tests/js/spec/utils/profiling/profile/profile.spec.tsx

```diff
@@ -0,0 +1,150 @@
+import {CallTreeNode} from 'sentry/utils/profiling/callTreeNode';
+import {Frame} from 'sentry/utils/profiling/frame';
+import {Profile} from 'sentry/utils/profiling/profile/profile';
+
+const f = (name: string, key: number) => new Frame({name, key, is_application: false});
+const c = (fr: Frame) => new CallTreeNode(fr, null);
+
+const makeTestingBoilerplate = () => {
+  const timings: [string, string][] = [];
```

Comment:
This type can be further refined right? The first tuple is `frame.name`. Specifying the types makes it the rest of the func easier to understand imo